src/components/variantsListPanel/VariantsPanelList.vue

```diff
@@ -27,6 +27,11 @@ import BaseVariantIcon from './BaseVariantIcon.vue';
 
 export default {
```

Comment:
There are eslint errors, you can check it by command: npm run lint, fix it please